spec/features/admins/admin_users_index_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+
```

Comment:
Add spec to check if admin can see a users index and that only the admins can access the users index